.github/workflows/automatic-doc-checks.yml

```
@@ -0,0 +1,16 @@
+name: Main Documentation Checks
+
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
```

As discussed, I think we should just go for manual runs for now.
```suggestion
  - workflow_dispatch
```